### DISCUSSION TITLE
Pinned

### DISCUSSION THREAD

**Discussion by bomabomaboma** [xenia-canary/game-patches#19] [General]
Currently there are no obvious links or even a mention about patches on Canary readme. It's difficult to access, or even to be aware that these patches exist beyond mentions in Youtube videos. Is that the most the team wants to push these right now?

Questions about patches are quickly becoming one of the most asked questions in the discord. I get DMs of so many people troubleshooting patches with me.

I'm just wondering if we want to pin a thread at the top of the Reddit, or fix up the Canary Wiki, which I would be happy to do. The last five years of my professional experience has been far more weighed toward design & presentations, and the myriad of apis and software involved.

But if you guys are happy with how things are right now, that's totally cool too. Merely my opinion.

**Comment by illusionyy** [comment]
I'll wait until the patching system is merged into mainline
